pkg/controller/cluster/pod.go

```diff
@@ -59,7 +59,7 @@ func AddPodController(ctx context.Context, mgr manager.Manager) error {
 }
 
 func (p *PodReconciler) Reconcile(ctx context.Context, req reconcile.Request) (reconcile.Result, error) {
-	log := ctrl.LoggerFrom(ctx).WithValues("pod", req.NamespacedName)
+	log := ctrl.LoggerFrom(ctx).WithValues("statefulset", req.NamespacedName)
```

Comment:
Ah, so this is actually receiving StatefulSets because of the `EnqueueRequestForOwner`. Gotcha. :+1:

Reply:
yup